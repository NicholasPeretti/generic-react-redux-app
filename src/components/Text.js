import React from 'react'
import styled from 'styled-components'

const StyledSpan = styled.span`
  font-size: 14px;
  font-weight: normal;
  margin: 0px;
`

const Text = ({ children, style = {} }) => (
  <StyledSpan style={style}>
    {children}
  </StyledSpan>
)

export default Text
