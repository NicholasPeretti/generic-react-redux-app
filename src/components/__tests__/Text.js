import React from 'react'
import { shallow } from 'enzyme'
import PresentationalTest from '../../utils/tests/presentational'
import Text from '../Text'

describe('Text', () => {
  PresentationalTest(Text)

  test('Should match snapshot', () => {
    const wrapper = shallow(<Text>This is a Text</Text>)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
