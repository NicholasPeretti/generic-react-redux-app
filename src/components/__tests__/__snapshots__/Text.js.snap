// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text Should match snapshot 1`] = `"<span class=\\"sc-bdVaJa bkmfbd\\">This is a Text</span>"`;
